service/header/service.go

```diff
@@ -44,3 +44,11 @@ func (s *Service) Stop(context.Context) error {
 	log.Info("stopping header service")
 	return nil
 }
+
+// TODO(@Bidon15): We need a sync subscription
+// for better handling of sync status
```

Comment:
can you make this into an issue instead of a code comment?

Reply:
I've left the TODO with the issue 
Is it ok?